.htmllintrc

```diff
@@ -0,0 +1,19 @@
+{
+	"indent-style": "tabs",
+	"line-end-style": false,
+	"attr-quote-style": "double",
```

Comment:
Is this needed, since `double` is the default value of `attr-quote-style`?

Reply:
removed. good catch. I just copied the `.htmllintrc` file from one of the other chart.js repos. I guess we could remove it from the others as well